qa-core/src/tests/internal-api/screens/screens.spec.ts

```diff
@@ -0,0 +1,71 @@
+import TestConfiguration from "../../../config/internal-api/TestConfiguration"
+import { App } from "@budibase/types"
+import InternalAPIClient from "../../../config/internal-api/TestConfiguration/InternalAPIClient"
+import generateApp from "../../../config/internal-api/fixtures/applications"
+import { Screen } from  "@budibase/types"
+import generateScreen from "../../../config/internal-api/fixtures/screens"
+
+
+describe("Internal API - /screens endpoints", () => {
```

Comment:
A suggestion for how we could use a very predictable jest layout and naming convention for this and other test going forward:

```ts
  // this tells me the whole root endpoint, and tells me it's internal api
  describe("/api/screens", () => {

    // we incorrectly use post on the root endpoint for both create and update
    // so we could add a distinguishing suffix in some cases
    it("POST - create", () => {})
    it("POST - update", () => {})

    it("GET", () => {})

    // this tells me I am working on a particular screen
    describe("/api/screens/:id/:rev", () => {
      it("DELETE", () => {})
    })

  })
```

The basic idea here being that each unique endpoint has it's own `describe` block and the `it` blocks within define the http method used. This actually highlights that our screen update behaviour is incorrect and should really be nested under `/api/screens/:id`